SR2019/2019-01-26-competition-date-announcement.md

```diff
@@ -0,0 +1,26 @@
+---
+to: Student Robotics 2019
+subject: Confirming the dates for Student Robotics 2019
+---
+
+Hello all,
+
+As some eagle-eyed members of your team may have noticed on [the forum](https://studentrobotics.org/forum/viewtopic.php?f=9&t=35), today's the day we confirm the dates for Student Robotics 2019!
+
+The competition will be held from *Saturday 6th April* until *Sunday 7th April*. The competition will be held at the University of Southampton's Highfield campus, so please plan accommodation / transport accordingly. The exact start/end times are to be confirmed, but doors open 9am, and we aim to finish around 5pm.
```

Comment:
We usually run later than 5pm. 2017's schedule has us running to 17:30 on the Saturday and 18:00 on the Sunday.
I appreciate we might want to re-jig to have a longer Saturday, though I anticipate we'll still want to run later than 5pm on at least one day.

Maybe say we'll aim to finish by 6pm?